Doc/library/profile.rst

```diff
@@ -577,23 +584,20 @@ implementations.
 Limitations
 ===========
 
-One limitation has to do with accuracy of timing information. There is a
-fundamental problem with deterministic profilers involving accuracy.  The most
-obvious restriction is that the underlying "clock" is only ticking at a rate
-(typically) of about .001 seconds.  Hence no measurements will be more accurate
-than the underlying clock.  If enough measurements are taken, then the "error"
-will tend to average out. Unfortunately, removing this first error induces a
-second source of error.
+To avoid infinite recursion, :mod:`profile` and :mod:`cProfile` do not work
+inside the trace function set by :func:`sys.settrace` or :func:`sys.setprofile`.
```

Comment:
```suggestion
inside trace functions set using :func:`sys.settrace` or :func:`sys.setprofile`.
```

Reply:
I'd also move this down near the bottom, since it seems like a pretty niche issue.